UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -441,5 +472,36 @@ private async Task CreateLargeMediaExchangesetCatalogFile(string batchId, string
             await Task.WhenAll(ParallelCreateFolderTasks);
             ParallelCreateFolderTasks.Clear();
         }
+
+        private async Task<bool> PackageAndUploadLargeMediaExchangeSetZipFileToFileShareService(string batchId, string exchangeSetPath, string exchangeSetZipFilePath, string correlationId, string mediaZipFileName)
+        {
+            bool isZipFileUploaded = false;
+            bool isZipFileCreated = false;
+            DateTime createZipFileTaskStartedAt = DateTime.UtcNow;
+
+            isZipFileCreated = await logger.LogStartEndAndElapsedTimeAsync(EventIds.CreateZipFileRequestStart,
+                       EventIds.CreateZipFileRequestCompleted,
+                       "Create large media exchange set zip file request for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}",
+                       async () =>
+                       {
+                           return await fulfilmentFileShareService.CreateZipFileForExchangeSet(batchId, exchangeSetPath, correlationId);
+                       },
+                       batchId, correlationId);
+
+            DateTime createZipFileTaskCompletedAt = DateTime.UtcNow;
```

Comment:
May as well just pass DateTime.UtcNow directly in the parameters?

Reply:
Resolved